nodejs/common/bin/NoGame/Common/ExperienceCalculator.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
+
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+
+var Assert = require('assert-js');
+
+var ExperienceCalculator = function () {
```

Comment:
why logic/calculations from this class are duplicated in https://github.com/norzechowicz/no-game/blob/ed36e0327309e164c273b8b143eb418461bd049d/nodejs/common/src/NoGame/Common/ExperienceCalculator.js ?